src/BingoGame.Indexer.CA/GraphQL/BingoResultDto.cs

```diff
@@ -0,0 +1,50 @@
+using AElf.Indexing.Elasticsearch;
+using AElfIndexer.Client;
+using GraphQL;
+using Nest;
+using BingoGame.Indexer.CA.GraphQL;
+
+namespace BingoGame.Indexer.CA.Entities;
```

Comment:
The namespace doesn't align with the folder name. Please fix it.

Reply:
noted, changed Entities to GraphQL